Rakefile

```diff
@@ -70,3 +70,41 @@ task :populate, [:storage_root] => [:environment] do |_t, args|
     puts "You need to enter a specific storage root"
   end
 end
+
+desc "Fire off M2C existence check on a single storage root"
+task :m2c_exist_single_root, [:storage_root, :boolean, :profile] => [:environment] do |_t, args|
+  unless args[:profile] == 'profile' || args[:profile].nil?
+    p "Usage: rake m2c_exist_single_root[fixture_sr1,false] || rake m2c_exist_single_root[fixture_sr1, false, profile]"
+    exit
+  end
+  root = args[:storage_root].to_sym
+  storage_dir = "#{Settings.moab.storage_roots[root]}/#{Settings.moab.storage_trunk}"
+  puts "#{Time.now.utc.iso8601} Running moab to catalog existence check for #{storage_dir}"
+  $stdout.flush # sometimes above doesn't end up getting flushed to STDOUT till the last puts when the run finishes
+  if args[:profile] == 'profile'
+    puts "When done, check log/profile_flat_check_existence_for_dir[TIMESTAMP].log for profiling details"
```

Comment:
is this also `.txt` or is this actually `.log` ?

Also - does this need correcting for seeding tasks?

Reply:
ah good catch.